.github/workflows/ConvertGuidelinesXmlToMarkdown.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches: 
       - 'master'
+  workflow_dispatch:
```

Comment:
Workflow dispatch fails on the `mudlabs/simple-diff` step. So if we want to do something with this we will need to consider alternatives.

Reply:
Yeah we learned that simple diff only works on commits etc...
So this workflow can now be removed as
I created another workflow in the master branch that adds the dispatch option. It was just a way to generate the markdown manually if working in branch other than the master and want to see the output.

Reply:
This is true, I just dropped this commit, there is another PR that should address this as Austin has mentioned.